docs/content/action/metadata/time_start.md

```diff
@@ -0,0 +1,23 @@
+---
+navtitle: time_start
+related:
+- /action/metadata/time_end
+---
+
+# Metadata "time_start"
+
+The `time_start` metadata stores a Unix timestamp of when the action started executing.
```

Comment:
The `time_start` metadata stores the Unix timestamp at the time when the action is executed.

Reply:
...of the time...